project/cmake/modules/FindIMXVPU.cmake

```diff
@@ -0,0 +1,27 @@
+# - Try to find IMXVPU
+# Once done this will define
+#
+# IMXVPU_FOUND - system has IMXVPU
+# IMXVPU_INCLUDE_DIRS - the IMXVPU include directory
+# IMXVPU_LIBRARIES - The IMXVPU libraries
+
```

Comment:
Please adapt as in https://github.com/xbmc/xbmc/commit/d8877a284c63350dd0cea25c0275a258f6cc4a4d.

XX_INCLUDE_DIRS and XX_LIBRARIES should not be cached vars (so they can't be used directly by find_path/library. Use XX_INCLUDE_DIR and XX_LIBRARY. The paths from package config should can hint find_path/library by setting PATHS. Docs: https://cmake.org/cmake/help/latest/manual/cmake-developer.7.html#find-modules